File: Leerjaar 2/Lesson5/Player.py
class Player:

    # constructor
    def __init__(self, name, level):
        self.name = name
        self.level = level

    # getter
    def get_name(self):
        return self.name

    # setter
    def set_name(self, name):
        self.name = name


player = Player('Joost', 20)

File: Leerjaar 2/Lesson5/Player.php
<?
class Player{
  private $name;
  private $level;

  // constructor
  function __construct($name, $level){
    $this->name = $name;
    $this->level = $level;
  }

  // getter
  function getName(){
    return $this->name;
  }

  // setter
  function setName($name){
    $this->name = $name;
  }
}

$player = new Player('joost', 20);